test/test_prototype_transforms.py

```diff
@@ -1395,10 +1394,9 @@ def test__get_params(self, mocker):
         assert isinstance(size, tuple) and len(size) == 2
 
         longer = max(size)
-        assert longer <= max_size
-
         shorter = min(size)
-        if longer == max_size:
+        if max_size is not None and longer == max_size:
```

Comment:
This is condition seems strange `and longer == max_size`

Reply:
Yeap, not sure why this existed before. Fixed. Still passes locally on multiple runs.